src/Extensions/azure-storage/azure-storage/Microsoft.Diagnostics.Monitoring.AzureStorage.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Remove the tabs. And the indentation should be 2 spaces for csproj files.

Reply:
Removed